src/observable.js

```diff
@@ -114,6 +114,52 @@ extend(Observable.prototype, {
     return this._off(ANY, fn);
   },
 
+  subscribe(observer) {
+    const _this = this;
+    let closed = false;
+
+    if (typeof observer === 'function') {
+      observer = {
+        next: observer,
+        error: arguments[1],
+        complete: arguments[2]
+      };
+    }
+
+    if (observer.next) {
+      this.onValue(observer.next);
```

Comment:
Yeah, I'm not a fun of this situation too. This is why I didn't want to use same name for the method originally. But I think this is still a reasonable trade of (with a note about different behavior in the docs + not having `Symbol.observable` on `Observable`).